mmv1/products/firestore/terraform.yaml

```diff
@@ -13,6 +13,41 @@
 
 --- !ruby/object:Provider::Terraform::Config
 overrides: !ruby/object:Overrides::ResourceOverrides
+  Database: !ruby/object:Overrides::Terraform::ResourceOverride
+    skip_delete: True
+    autogen_async: True
+    id_format: "projects/{{project}}/databases/{{name}}"
+    import_format:
+      - "projects/{{project}}/databases/{{name}}"
+      - "{{project}}/{{name}}"
+      - "{{name}}"
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "firestore_database_default"
```

Comment:
Consider removing `default`. In Terraform layer, database_id is transparent so `(default)` vs `test-db` do not make any difference.

Reply:
I think I'll leave this in for now. I'd like to emphasize that this example is about creating the (default) database.

Reply:
On second thought, you're right, there's nothing too special about this. I'll change the name.